src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -14,6 +14,7 @@ import IconChatClockGrey from '../../../assets/icon/IconChatClockGrey';
 import {ChatItemMyTextProps} from '../../../../types/component/AnonymousChat/BaseChatItem.types';
 import {ChatStatus} from '../../../../types/database/schema/ChannelList.types';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
+import {SIGNED} from '../../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `SIGNED` is added but it's not clear where this constant is used in the code. If it's not used, it should be removed to avoid unnecessary imports.
```diff
- import {SIGNED} from '../../../hooks/core/constant';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->